sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/auth/SimpleWebAuthServiceImpl.java

```diff
@@ -25,8 +25,8 @@
  * @author cdfive
  * @since 1.6.0
  */
-@Primary
 @Component
+@ConditionalOnProperty(name="auth.enabled", havingValue="true")
 public class SimpleWebAuthServiceImpl implements AuthService<HttpServletRequest> {
```

Comment:
There is a property `matchIfMissing` in `@ConditionalOnProperty`, default is false, maybe we can set it true here
in order to make it as default when no `auth.enabled` added in application.properties.

Besides, suggest that add a space between `=` for better code formatting.

Reply:
Yes